### ISSUE TITLE
Spaces before and after tags are being stripped from paragraphs 

### ISSUE THREAD

**Issue by matt-quoin** [Aymkdn/html-to-pdfmake#9]
After updating to version 1.0.6, parsing paragraphs that have em, strong, etc. tags lose the spaces before and after them. This looks to be broken on the default text in the converter too: https://aymkdn.github.io/html-to-pdfmake/index.html

**Comment by Aymkdn** [comment]
Yes I added a trim because it seemed useless to keep empty spaces like that. What is the issue with that?

**Comment by Aymkdn** [comment]
If you have an example of a problematic code to let me understand what's going on?

**Comment by matt-quoin** [comment]
Thanks for the quick reply! An example would be this HTML:
```
<p>
    This is a sentence with a <strong>bold word</strong>, <em>one in italic</em>,
    and <u>one with underline</u>.
</p>
```

which converts to:

```
[
  {
    "text": [
      {
        "text": "This is a sentence with a"
      },
      {
        "text": "bold word",
        "bold": true,
        "style": [
          "html-strong"
        ]
      },
      {
        "text": ","
      },
      {
        "text": "one in italic",
        "italics": true,
        "style": [
          "html-em"
        ]
      },
      {
        "text": ", and"
      },
      {
        "text": "one with underline",
        "decoration": "underline",
        "style": [
          "html-u"
        ]
      },
      {
        "text": "."
      }
    ],
    "margin": [
      0,
      5,
      0,
      10
    ],
    "style": [
      "html-p"
    ]
  }
]
```

which looks like this in the pdf:
<img width="382" alt="Screen Shot 2019-08-30 at 8 59 25 AM" src="https://user-images.githubusercontent.com/8090999/64022594-8ef15880-cb04-11e9-8989-27d268bfdab3.png">

The spaces around the words in bold, italics, or underlined get removed so the word 'a' is now attached to the word 'bold', etc.

**Comment by Aymkdn** [comment]
I see. But, too late, I'm already gone on vacation....

While waiting for me to get back, you can remove `. trim()` on https://github.com/Aymkdn/html-to-pdfmake/blob/9c49074dd9cefcd0491bbf0f84fb76eba1613139/index.js#L94

When I'm back I'll release a new version with a fix.

**Comment by Aymkdn** [comment]
That should be resolved with v1.0.7
